pvnet_app/app.py

```diff
@@ -217,6 +224,104 @@ def filter(self, record):
 # APP MAIN
 
 
+def get_dataloader(config_filename: str, t0: pd.Timestamp, gsp_ids: list[int], num_workers: int):
```

Comment:
could you put these in a file called dataloader?

Reply:
I prefer having those legacy things up front and in the way so that they aren't just forgotten about

Reply:
sorry, i mean move all the dataloader things into a seperate file

Reply:
Then its very clear that the dataloader code is in that file. Where as app.py, its not directly obviously if dataloader is in that code or somewhere else. So its just for readibilty